### ISSUE TITLE
Cannot disable animations.

### ISSUE THREAD

**Issue by orbziiharry** [georstat/react-native-image-cache#53]
Attempting to set the animation duration to 0 to essentially disable animations doesn't work because the check for the passed prop is checking for a falsey value not for undefined so `0` is counted as false.

here for the thumbnail: https://github.com/georstat/react-native-image-cache/blob/main/src/CachedImage.tsx#L137

and here for the main image: https://github.com/georstat/react-native-image-cache/blob/main/src/CachedImage.tsx#LL150C5-L155C8

This issue alsoa pplies for the cache manager config

**Comment by bisak** [comment]
Yup, I'm running into the same issue :)

**Comment by orbziiharry** [comment]
Setting the animation duration to 1ms is a sort of workaround that’s not noticeable.

**Comment by rodgomesc** [comment]
```diff
diff --git a/node_modules/@georstat/react-native-image-cache/src/CachedImage.tsx b/node_modules/@georstat/react-native-image-cache/src/CachedImage.tsx
index bf1a417..f8323c5 100644
--- a/node_modules/@georstat/react-native-image-cache/src/CachedImage.tsx
+++ b/node_modules/@georstat/react-native-image-cache/src/CachedImage.tsx
@@ -131,27 +131,31 @@ const CachedImage = (props: IProps & typeof defaultProps) => {
   };
 
   const onThumbnailLoad = () => {
+    const thumbnailAnimationDuration =
+      props.thumbnailAnimationDuration !== undefined
+        ? props.thumbnailAnimationDuration
+        : CacheManager?.config?.thumbnailAnimationDuration || 100;
+
     animatedLoadingImage.value = withTiming(0, {}, () => {
       animatedThumbnailImage.value = withTiming(1, {
-        duration:
-          props.thumbnailAnimationDuration ||
-          CacheManager?.config?.thumbnailAnimationDuration ||
-          100,
+        duration: thumbnailAnimationDuration,
       });
     });
-  };
+};
 
   const onImageError = (): void => setError(true);
 
   const onImageLoad = (e: NativeSyntheticEvent<ImageLoadEventData>): void => {
+    const sourceAnimationDuration =
+      props.sourceAnimationDuration !== undefined
+        ? props.sourceAnimationDuration
+        : CacheManager?.config?.sourceAnimationDuration || 1000;
+  
     if (props.onLoad) {
       props.onLoad(e);
     }
     animatedImage.value = withTiming(1, {
-      duration:
-        props.sourceAnimationDuration ||
-        CacheManager?.config?.sourceAnimationDuration ||
-        1000,
+      duration: sourceAnimationDuration,
     });
   };
 ```

**Comment by efstathiosntonas** [comment]
Hey folks, sorry for the delay on this one. I've forgotten to fix it on `2.6.0` published earlier today, the fix is now available on `2.7.0`: https://github.com/georstat/react-native-image-cache/releases/tag/v2.7.0

**Comment by orbziiharry** [comment]
Great new @efstathiosntonas thank you!
